tsconfig.json

```diff
@@ -27,6 +27,13 @@
     "resolveJsonModule": true,
     "isolatedModules": true
   },
-  "include": ["src"],
-  "exclude": ["node_modules", "dist", "example"]
+  "include": ["src/**/*.ts", "src/**/*.tsx"],
```

Comment:
Should we also include `"src/*.ts", "src/*.tsc",` ?

Reply:
We don't want to include everything, per-se. Having `setupTests.d.ts` wouldn't be particularly useful